bin/chrome2calltree.js

```diff
@@ -30,6 +30,10 @@ parser.addArgument(['-k', '--kcachegrind'], {
             "remove it after kcachegrind exits.",
     'action': 'storeTrue'
 });
+parser.addArgument(['-m', '--mutate'], {
```

Comment:
Oh, it's fine for this to be default. I can't imagine that you'd ever want the command line util to do the slower thing.

When I said make it default to not mutating, I just meant for chromeProfileToCallgrind, since people may be consuming that as a library.

You can just remove this flag completely, and say `copy = false` below in the `readEntireStream` callback below.